profiles/Samarth-Saxena.json

```
@@ -0,0 +1,6 @@
+{
+    "image":"https://user-images.githubusercontent.com/90990378/211378645-134aba56-6085-4db4-a02b-61d07fc275dd.png",
+    "issueId": 1634,
+    "name": "Samarth Saxena",
+    "username": "SamarthSaxena10"
+  }
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```